RIAssigner/data/MatchMSData.py

```diff
@@ -153,4 +184,6 @@ def _assign_ri_value(spectrum: Spectrum, key: str, value: Data.RetentionIndexTyp
     """
     if value is not None:
         retention_index = ('%f' % float(value)).rstrip('0').rstrip('.')
+        if key == 'retentionindex':
+            key = 'retention_index'
         spectrum.set(key=key, value=retention_index)
```

Comment:
Why is this needed?

Reply:
It is not needed. It was supposed to fix an error that appeared before unit test  implementation. Reverted to original ([f90fd71](https://github.com/RECETOX/RIAssigner/pull/112/commits/f90fd712eae831b7a43c393096a557bc81a35ea4))